src/components/navigation/NavigationKeyHandler.ts

```diff
@@ -7,32 +7,51 @@
 
 import { wiper } from "../../utils/wiper";
 import { addDataNavigation } from "./AddNavigation";
+import { jumpXcharts, jumpXelements } from "./JumpX";
+import { skip } from "./Skip";
+import { xSetter } from "./XSetter";
 
 /**
  * Listens for navigation related keypresses and handles the outcomes.
  *
  * @export
- * @param {React.KeyboardEvent<HTMLDivElement>} event
+ * @param {string} type
+ * @param {React.KeyboardEvent} event
+ * @param {number} number
  * @param {React.RefObject<HTMLElement>} ref
- * @param {HTMLElement} [alertDiv]
- * @param {{ element?: string; className?: string }} [selectorType]
+ * @param {*} elements
+ * @param {HTMLDivElement} alertDiv
+ * @param {string} selectedSeries
+ * @param {string[]} series
+ * @param {{ element?: string; className?: string }} selectorType
  * @param {(string | undefined)} [multiSeries]
  * @param {() => void} [nextSeries]
- * @param {string[]} [series]
- * @param {string} [selectedSeries]
- * @return {void}
+ * @return {number} Number of points being jumped at a time inside the wrapped chart.
  */
-export function levelNav(
-	event: React.KeyboardEvent<HTMLDivElement>,
+export function navigationKeyHandler(
```

Comment:
Read me comment above on the number of arguments.
The more arguments a function has, the less maintainable it becomes and more prone to errors.